components/server/src/server.ts

```diff
@@ -194,43 +199,10 @@ export class Server<C extends GitpodClient, S extends GitpodServer> {
         await this.registerRoutes(app);
 
         // Turn unhandled requests into errors
-        app.use((req: express.Request, res: express.Response, next: express.NextFunction) => {
-            if (this.isAnsweredRequest(req, res)) {
-                return next();
-            }
-
-            // As we direct browsers to *api*.gitpod.io/login, we get requests like the following, which we do not want to end up in the error logs
-            if (req.originalUrl === '/'
```

Comment:
are you sure, this is not introducing new noise?

Reply:
@AlexTugarev IMO we no longer use `api.gitpod.io`. We did the switch to `gitpod.io/api` ~1,25 years ago. And `Caddyfile` does no longer contain `api.gitpod.io` at all, so I deem this to be safe. :+1: